### ISSUE TITLE
Add window title and icon

### ISSUE THREAD

**Issue by yungyuc** [solvcon/solvcon#58]
The viewer does not yet have window title and icon.

**Comment by yanagiragi** [comment]
Shouldn't be hard. Please let me have a try.

**Comment by yungyuc** [comment]
The macos icon is not available even after #92.  It can be handled in a separate PR.

**Comment by yungyuc** [comment]
At the present time modmesh has been enhanced to be able to launch the GUI window from a standard Python interpreter (instead of starting from the standalone `viewer` binary).

We should also clarify whether or not it is possible to add window icon when the window is started from a standard interpreter.  If it's possible, the feature should be added.
